examples/ecmascript/specs/features/using-docstrings.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Certificate manufaturing
```

Comment:
Looks like a typo: manufacturing is missing the "c".

Reply:
Indeed :)

Reply:
fixed